joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50 

joint2_controller:
  type: "position_controllers/JointTrajectoryController"
  joints: 
   - joint2
gazebo_ros_control:
  pid_gains:
      joint2:
          p: 100.0
          i: 1.0
          d: 0.1